<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(100, 100, 100);</string>
  </property>
  <widget class="QPushButton" name="gotomenubutton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 80, 80);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Return to Menu</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="PetlistWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>451</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>My Pets</string>
    </property>
   </column>
  </widget>
  <zorder>PetlistWidget</zorder>
  <zorder>gotomenubutton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
